orchestra/admin/__init__.py

```diff
@@ -7,6 +7,7 @@
 from orchestra.models import Step
 from orchestra.models import Task
 from orchestra.models import TaskAssignment
+from orchestra.models import TimeEntry
```

Comment:
Yep! We break imports into three sections based on their source: stdlib, third-party, and internal (sometimes more for more granularity, e.g., if there's a lot of `django` imports). Each section looks like this:
```
import <module>
from <module> import <lowercase_name>
from <module> import <uppercase_name>
```